iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/DanglingLineXml.java

```diff
@@ -31,7 +33,19 @@ protected String getRootElementName() {
 
     @Override
     protected boolean hasSubElements(DanglingLine dl) {
-        return dl.getCurrentLimits() != null || dl.getGeneration() != null;
+        return false;
```

Comment:
It's a bit confusing to get a possibly different result if you forget to provide the context. Should we raise an exception? or maybe add a small comment?

Reply:
Yes, you're right I will replace it by an `AssertionError`, the method should not be called